_includes/releases/v22.1/v22.1.0-alpha.5.md

```diff
@@ -0,0 +1,73 @@
+- CockroachDB now limits the number of concurrent catchup scan requests issued by [rangefeed](../v22.1/use-changefeeds.html#enable-rangefeeds) clients. [#77866][#77866]
+
+<h3 id="v22-1-0-alpha-5-sql-language-changes">SQL language changes</h3>
+
+- TTL metrics are now labelled by relation name if the `server.child_metrics.enabled` [cluster setting](../v22.1/cluster-settings.html) is enabled and the `ttl_label_metrics` storage parameter is set to `true`. This is to prevent a potentially unbounded cardinality on TTL related metrics. [#77567][#77567]
+- Added support for the `MOVE` command, which moves a SQL cursor without fetching any rows from it. `MOVE` is identical to [`FETCH`](../v22.1/limit-offset.html), including in its arguments and syntax, except it doesn't return any rows. [#74877][#74877]
+- Added the `enable_implicit_transaction_for_batch_statements` [session variable](../v22.1/set-vars.html). It defaults to false. When true, multiple statements in a single query (a "batch statement") will all run in the same implicit transaction, which matches the PostgreSQL wire protocol. This setting is provided for users who want to preserve the behavior of CockroachDB versions v21.2 and earlier. [#77865][#77865]
+- The `enable_implicit_transaction_for_batch_statements` session variable now defaults to false. [#77973][#77973]
+- The `experimental_enable_hash_sharded_indexes` session variable is deprecated as hash sharded indexes are enabled by default. Enabling this setting results in a noop. [#78038][#78038]
+- New `crdb_internal.merge_stats_metadata` [builtin function](../v22.1/functions-and-operators.html#built-in-functions) to group statement statistics metadata. [#78064][#78064]
+- [Changefeeds](../v22.1/use-changefeeds.html) can now specify column families to target, using the syntax `[TABLE] foo FAMILY bar`. For example, `CREATE CHANGEFEED FOR TABLE foo FAMILY bar, TABLE foo FAMILY baz, TABLE users` will create a feed that watches the `bar` and `baz` column families of `foo`, as well as the whole table `users`. A family must exist with that name when the feed is created. If all columns in a watched family are dropped in an `ALTER TABLE` statement, the feed will fail with an error, similar to dropping a table. The behavior is otherwise similar to feeds created using `split_column_families`. [#77964][#77964]
+- [Casts](../v22.1/data-types.html#data-type-conversions-and-casts) that are affected by the `DateStyle` or `IntervalStyle` session variables used in [computed columns](../v22.1/computed-columns.html) or [partial index](../v22.1/partial-indexes.html) definitions will be rewritten to use immutable functions after upgrading to v22.1. This is in preparation for eventually allowing `DateStyle` and `IntervalStyle` to be used by default. Currently they are gated behind the `datestyle_enabled` and `intervalstyle_enabled` cluster settings. [#78229][#78229]
```

Comment:
I'm not sure we should include the last 2 sentence as they are forward looking and not actionable now.